scripts/deploy/nlp/deploy_triton.py

```diff
@@ -378,11 +225,7 @@ def get_nemo_deployable(args):
 def nemo_deploy(argv):
     args = get_args(argv)
```

Comment:
We don't need to pass `argv` around

Reply:
I mean that we can just write `nemo_deploy()` in [L291](https://github.com/NVIDIA/NeMo/blob/896eb168e028470085c8092159c24f56bdf0a37e/scripts/deploy/nlp/deploy_triton.py#L291) to further simplify this as `sys.argv[1:]` is used by default for parsing arguments